charts/careers/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. check that the application deployed well by running the following commands
```

Comment:
Did you want to remove this or intentional to keep it in here?

Reply:
I thought it'd be nice to leave in.  TBH I was on the fence.  Which way would you lean @bkochendorfer ?